### PR TITLE
Creation of authentication methods and call added to screens.dart

### PR DESCRIPTION
**Pull request by Neechie**
Import call in loginScreen.dart and createAccountScreen.dart are not yet functional and will need some clarification on that process

### CODE REVIEW

**Comment by JPyke3** [comment]
When you're happy with the changes I made @Neechie, ill merge and close the branch.

File: lib/models/auth.dart [hunk 1]
```diff
@@ -6,23 +6,26 @@ import 'package:firebase_auth/firebase_auth.dart';
  Last thing i did was move the file into the Models folder, figured it would make more sense
 */
 
-
+/// The authentication Class, used for logging in users.
```
Comment (JPyke3) [inline lib/models/auth.dart:9]: Just so you know, any comments that have a triple slash will appear when moused over in an IDE, useful for when working as a team.

Comment (JPyke3) [reply lib/models/auth.dart:9]: @Neechie

**Review by JPyke3 — CHANGES_REQUESTED** [review]
Review Code and comment if you're happy to go ahead.

**Comment by Neechie** [comment]
Thanks @JPyke3, those changes all make sense to me! Thanks for looking over it. Merge and close when you are ready

**Review by JPyke3 — APPROVED** [review]

**Pull request merged**